README.md

```diff
@@ -117,6 +117,71 @@ python -m build
 # 📈 Training
 
 The torch-ort library accelerates training of large transformer PyTorch models to reduce the training time and GPU cost with a few lines of code change. It is built on top of highly successful and proven technologies of ONNX Runtime and ONNX format and includes the ONNX Runtime Optimizer and Data Sampler.
+## Add ONNX Runtime for PyTorch to your PyTorch training script
+
+```python
+from torch_ort import ORTModule
+model = ORTModule(model)
+# PyTorch training script follows
+```
+
+## Usage of FusedAdam and FP16 Optimizer (Optional)
+
```

Comment:
Shall we give an intro on what FP16_Optimizer is?

Reply:
I think we can add a link to fp16_optimizer.py like FusedAdam.

Reply:
if possible, let's add a line or two introducing what it does and then ask users to read FP16_Optimizer.py for more details.